charts/super-agent-deployment/README.md

```diff
@@ -0,0 +1,86 @@
+[![Community Plus header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Plus.png)](https://opensource.newrelic.com/oss-category/#community-plus)
+
+# super-agent-deployment
+
+![Version: 0.0.0-beta](https://img.shields.io/badge/Version-0.0.0--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: TODO-FILL-THIS-WITH-CI](https://img.shields.io/badge/AppVersion-TODO--FILL--THIS--WITH--CI-informational?style=flat-square)
+
+A Helm chart to install New Relic Super agent on Kubernetes
+
+# Helm installation
+
+You can install this chart using directly this Helm repository:
+
+```shell
+helm repo add newrelic https://newrelic.github.io/helm-charts
+helm upgrade --install newrelic/CHART-TEMPLATE -f your-custom-values.yaml
```

Comment:
CHART-TEMPLATE?

Reply:
I expected that people would complain about the chart names. I was wrong so I changed the URL to the definitive one on [#d0c2eb5](https://github.com/newrelic/helm-charts/pull/1155/commits/d0c2eb57da5a26394ec0e5e59e173ac18425c5fb)